code/modules/organs/internal/_internal.dm

```diff
@@ -171,7 +171,7 @@
 
 		// We clamp/round here so that we don't accidentally heal past the threshold and
 		// cheat our way into a full second threshold of healing.
-		damage = clamp(damage-get_organ_heal_amount(), min_heal_val, absolute_max_damage)
+		damage = clamp(damage - max(0, get_organ_heal_amount() * GET_CHEMICAL_EFFECT(owner, CE_BRAIN_REGEN)), min_heal_val, absolute_max_damage)
```

Comment:
doesn't this apply to all organs and not just the brain? i'm pretty sure this turns fish oil/neuroannealer into budget peridaxon. also it means it won't regenerate at all without any in your system, you might want like. `* (1 + GET_CHEMICAL_EFFECT(owner, CE_BRAIN_REGEN))` or something

if that's not desired my proposal would be to give organs a var like `healing_chem_effect` and do something like
```dm
		var/healing_multiplier = 1 + (healing_effect ? GET_CHEMICAL_EFFECT(owner, healing_effect) : 0)
		damage = clamp(damage - max(0, get_organ_heal_amount() * healing_multiplier), min_heal_val, absolute_max_damage)
```